CONTRIBUTING.md

```diff
@@ -11,6 +11,7 @@
       - [Rule type](#rule-type)
   - [Coding Guidelines](#coding-guidelines)
     - [C++](#c)
+    - [Unit testing](/tests/README.md)
```

Comment:
Do we want to write a section about tests in the [organization contributing file](https://github.com/falcosecurity/.github/blob/master/CONTRIBUTING.md) too?